runtime/compiler/runtime/RelocationRecord.hpp

```diff
@@ -1491,6 +1491,20 @@ class TR_RelocationRecordValidateDefiningClassFromCP : public TR_RelocationRecor
       virtual int32_t bytesInHeaderAndPayload() { return sizeof(TR_RelocationRecordValidateDefiningClassFromCPBinaryTemplate); }
       virtual void preparePrivateData(TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget) {}
       virtual int32_t applyRelocation(TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget, uint8_t *reloLocation);
+
+      virtual void print(TR_RelocationRuntime *reloRuntime);
+
+      void setiSStatic(TR_RelocationTarget *reloTarget, bool isStatic);
```

Comment:
last one :) . `setIsStatic()`

Reply:
Fixed in https://github.com/eclipse/openj9/pull/10298/commits/10a0d9da2b8910f8be1e86aed05ae3bad0d7211c.